recipes/console_bridge/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "console_bridge" %}
+{% set version = "0.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ros/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 9f024a38f0947ed9fa67f58829980c2d90d84740e6de20d75cb00866f07a7a0b
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+
+test:
+  requires:
+    - pkg-config
+  commands:
+    - pkg-config --modversion {{ name }}
+    - ls $PREFIX/lib/*{{ name }}*
```

Comment:
This is not actually testing anything, can you use `test -f $PREFIX/lib/lib{{ name }}${SHLIB_EXT}` or similar here?

Reply:
I pushed a commit doing this.